testsrc/test262.properties

```diff
@@ -1309,7 +1301,36 @@ built-ins/Promise 405/599 (67.61%)
 
 ~built-ins/Proxy
 
-~built-ins/Reflect
+built-ins/Reflect 29/139 (20.86%)
```

Comment:
For the failing Reflect tests (besides the ones failing because they rely on unsupported features like Proxy and computed-property-names or due to being strict mode related): why are these failing?

Reply:
these are mainly setter tests, from a first look they fail because i found no good way to provide the correct return value (false) if the setter is 'ignored'.
At least i found no way to do it without changing the current impl.

But i think 20% failing are 80% passing and so it's worth to start with this.....

Reply:
I think it matters why they fail: if those tests fail because of Reflect exhibiting wrong behavior it's a bigger issue that when they fail because something is not supported and errors out.

Because if it exhibits the wrong behavior, people might start to depend on it and thus fixing it becomes sort of a breaking change down the road